src/migrations/20220111194358-add-goals-objective-flag.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  up: async (queryInterface) => {
+    await queryInterface.sequelize.query(`
+      DO $$ BEGIN
+        ALTER TYPE "enum_Users_flags" ADD VALUE 'recipient_goals_objectives';
+      EXCEPTION
+        WHEN duplicate_object THEN null;
+      END $$;
+    `);
+  },
+
+  down: async () => {
+    // Is there a way to revert this? Not sure
```

Comment:
https://www.postgresql.org/docs/current/datatype-enum.html section 8.7.4. Removing existing values from the enum is not supported. To remove the old values we'd have to drop the type and add it again.